discovery-provider/alembic/versions/042c55d0efda_removing_is_ready_from_user_table.py

```diff
@@ -0,0 +1,26 @@
+"""removing is_ready from User table
+
+Revision ID: 042c55d0efda
+Revises: 7b3f27e78b84
+Create Date: 2020-06-01 21:29:35.777434
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '042c55d0efda'
+down_revision = '7b3f27e78b84'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column('users', 'is_ready')
+
+
+def downgrade():
```

Comment:
https://medium.com/the-andela-way/alembic-how-to-add-a-non-nullable-field-to-a-populated-table-998554003134